syntax = "proto3";

package auth;

message UserCreatedEvent {
  string userId = 1;
  string email = 2;
  int64 timestamp = 3;
  string urlToken = 4; 
}